modules/aggregation-layers/src/gpu-grid-layer/gpu-grid-cell-layer.js

```diff
@@ -53,9 +54,14 @@ export default class GPUGridCellLayer extends Layer {
   initializeState() {
```

Comment:
Should this layer be extending the base `GridCellLayer`?

Reply:
Attribute setup, state change detection, uniforms and shaders are completely different for both `GridCellLayer` and `GPUGridCellLayer` hence not extending from `GridCellLayer`.

Reply:
I will port recent elevation calculation changes from Column layer to GPUGridCellLayer in a separate PR.